app/code/Magento/LoginAsCustomer/Test/Mftf/ActionGroup/StorefrontAssertStickyLoginAsCustomerNotificationBannerActionGroup.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontAssertStickyLoginAsCustomerNotificationBannerActionGroup">
+        <annotations>
+            <description>Verify Sticky Login as Customer notification banner present on page.</description>
+        </annotations>
+        <arguments>
+            <argument name="customerFullName" type="string"/>
+            <argument name="websiteName" type="string" defaultValue="Main Website"/>
+        </arguments>
+
+        <waitForElementVisible selector="{{StorefrontLoginAsCustomerNotificationSection.notificationText}}" stepKey="waitForNotificationBanner"/>
+        <see selector="{{StorefrontLoginAsCustomerNotificationSection.notificationText}}"
+             userInput="You are connected as {{customerFullName}} on {{websiteName}}"
+             stepKey="assertCorrectNotificationBannerMessage"/>
+        <seeElement selector="{{StorefrontLoginAsCustomerNotificationSection.closeLink}}"
+             stepKey="assertCloseNotificationBannerPresent"/>
+        <executeJS function="window.scrollTo(0,document.body.scrollHeight);" stepKey="scrollToBottomOfPage"/>
```

Comment:
Have you tried to use `scrollTo` action instead of `executeJS`? You may still use offset values there.

Reply:
Hello @rogyar ...
Yes, I tried but I'm not able to find how we can scroll bottom of the page using MFTF cause I just find `scrollToTop` but not able to find any solution for scroll to bottom if you have any other solution then please share with us
Thanks

Reply:
As far as I know, there's no solution for this case "out of the box"